### ISSUE TITLE
2025-02-25, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#94]
# Today's Arxiv Papers

(1) [Introducing Visual Perception Token into Multimodal Large Language Model](http://arxiv.org/abs/2502.17425v1), 2025-02-24

(2) [Emergent Misalignment: Narrow finetuning can produce broadly misaligned LLMs](http://arxiv.org/abs/2502.17424v1), 2025-02-24

(3) [MLLMs Know Where to Look: Training-free Perception of Small Visual Details with Multimodal LLMs](http://arxiv.org/abs/2502.17422v1), 2025-02-24

(4) [LongSpec: Long-Context Speculative Decoding with Efficient Drafting and Verification](http://arxiv.org/abs/2502.17421v1), 2025-02-24

(5) [The Geometry of Refusal in Large Language Models: Concept Cones and Representational Independence](http://arxiv.org/abs/2502.17420v1), 2025-02-24

(6) [From System 1 to System 2: A Survey of Reasoning Large Language Models](http://arxiv.org/abs/2502.17419v1), 2025-02-24

(7) [Reasoning with Latent Thoughts: On the Power of Looped Transformers](http://arxiv.org/abs/2502.17416v1), 2025-02-24

(8) [COSMOS: A Hybrid Adaptive Optimizer for Memory-Efficient Training of LLMs](http://arxiv.org/abs/2502.17410v1), 2025-02-24

(9) [Linguistic Generalizability of Test-Time Scaling in Mathematical Reasoning](http://arxiv.org/abs/2502.17407v1), 2025-02-24

(10) [Large Language Models are Powerful EHR Encoders](http://arxiv.org/abs/2502.17403v1), 2025-02-24

(11) [Mitigating Bias in RAG: Controlling the Embedder](http://arxiv.org/abs/2502.17390v1), 2025-02-24

(12) [Big-Math: A Large-Scale, High-Quality Math Dataset for Reinforcement Learning in Language Models](http://arxiv.org/abs/2502.17387v1), 2025-02-24

(13) [What is a Good Question? Utility Estimation with LLM-based Simulations](http://arxiv.org/abs/2502.17383v1), 2025-02-24

(14) [Distributional Scaling Laws for Emergent Capabilities](http://arxiv.org/abs/2502.17356v1), 2025-02-24

(15) [On Relation-Specific Neurons in Large Language Models](http://arxiv.org/abs/2502.17355v1), 2025-02-24

(16) [How Scientists Use Large Language Models to Program](http://dx.doi.org/10.1145/3706598.3713668), 2025-02-24

(17) [Time series forecasting based on optimized LLM for fault prediction in distribution power grid insulators](http://arxiv.org/abs/2502.17341v1), 2025-02-24

(18) [Mutual Reinforcement of LLM Dialogue Synthesis and Summarization Capabilities for Few-Shot Dialogue Summarization](http://arxiv.org/abs/2502.17328v1), 2025-02-24

(19) [HIPPO: Enhancing the Table Understanding Capability of Large Language Models through Hybrid-Modal Preference Optimization](http://arxiv.org/abs/2502.17315v1), 2025-02-24

(20) [Child vs. machine language learning: Can the logical structure of human language unleash LLMs?](http://arxiv.org/abs/2502.17304v1), 2025-02-24

(21) [Delta Decompression for MoE-based LLMs Compression](http://arxiv.org/abs/2502.17298v1), 2025-02-24

(22) [Benchmarking Retrieval-Augmented Generation in Multi-Modal Contexts](http://arxiv.org/abs/2502.17297v1), 2025-02-24

(23) [Integrating protein sequence embeddings with structure via graph-based deep learning for the prediction of single-residue properties](http://arxiv.org/abs/2502.17294v1), 2025-02-24

(24) [Capability Instruction Tuning: A New Paradigm for Dynamic LLM Routing](http://arxiv.org/abs/2502.17282v1), 2025-02-24

(25) [MonoTODia: Translating Monologue Requests to Task-Oriented Dialogues](http://arxiv.org/abs/2502.17268v1), 2025-02-24

(26) [Unveiling Downstream Performance Scaling of LLMs: A Clustering-Based Perspective](http://arxiv.org/abs/2502.17262v1), 2025-02-24

(27) [Detecting Benchmark Contamination Through Watermarking](http://arxiv.org/abs/2502.17259v1), 2025-02-24

(28) [REINFORCE Adversarial Attacks on Large Language Models: An Adaptive, Distributional, and Semantic Objective](http://arxiv.org/abs/2502.17254v1), 2025-02-24

(29) [Alpha-SQL: Zero-Shot Text-to-SQL using Monte Carlo Tree Search](http://arxiv.org/abs/2502.17248v1), 2025-02-24

(30) [Baichuan-Audio: A Unified Framework for End-to-End Speech Interaction](http://arxiv.org/abs/2502.17239v1), 2025-02-24

(31) [Making LLMs Reason? The Intermediate Language Problem in Neurosymbolic Approaches](http://arxiv.org/abs/2502.17216v1), 2025-02-24

(32) [CoT-UQ: Improving Response-wise Uncertainty Quantification in LLMs with Chain-of-Thought](http://arxiv.org/abs/2502.17214v1), 2025-02-24

(33) [Order Matters: Investigate the Position Bias in Multi-constraint Instruction Following](http://arxiv.org/abs/2502.17204v1), 2025-02-24

(34) [IGDA: Interactive Graph Discovery through Large Language Model Agents](http://arxiv.org/abs/2502.17189v1), 2025-02-24

(35) [Evaluating Expert Contributions in a MoE LLM for Quiz-Based Tasks](http://arxiv.org/abs/2502.17187v1), 2025-02-24

(36) [Measuring Data Diversity for Instruction Tuning: A Systematic Analysis and A Reliable Metric](http://arxiv.org/abs/2502.17184v1), 2025-02-24

(37) [Cheems: A Practical Guidance for Building and Evaluating Chinese Reward Models from Scratch](http://arxiv.org/abs/2502.17173v1), 2025-02-24

(38) [Logic Haystacks: Probing LLMs Long-Context Logical Reasoning (Without Easily Identifiable Unrelated Padding)](http://arxiv.org/abs/2502.17169v1), 2025-02-24

(39) [JUREX-4E: Juridical Expert-Annotated Four-Element Knowledge Base for Legal Reasoning](http://arxiv.org/abs/2502.17166v1), 2025-02-24

(40) [MEMERAG: A Multilingual End-to-End Meta-Evaluation Benchmark for Retrieval Augmented Generation](http://arxiv.org/abs/2502.17163v1), 2025-02-24

(41) [Real-time Monitoring of Economic Shocks using Company Websites](http://arxiv.org/abs/2502.17161v1), 2025-02-24

(42) [Parameter Efficient Merging for Multimodal Large Language Models with Complementary Parameter Adaptation](http://arxiv.org/abs/2502.17159v1), 2025-02-24

(43) [CodeSwift: Accelerating LLM Inference for Efficient Code Generation](http://arxiv.org/abs/2502.17139v1), 2025-02-24

(44) [Evaluating the Effectiveness of Large Language Models in Automated News Article Summarization](http://arxiv.org/abs/2502.17136v1), 2025-02-24

(45) [Applications of Large Models in Medicine](http://dx.doi.org/10.71423/aimed.20250105), 2025-02-24

(46) [Thus Spake Long-Context Large Language Model](http://arxiv.org/abs/2502.17129v1), 2025-02-24

(47) [LettuceDetect: A Hallucination Detection Framework for RAG Applications](http://arxiv.org/abs/2502.17125v1), 2025-02-24

(48) [Adversarial Training for Defense Against Label Poisoning Attacks](http://arxiv.org/abs/2502.17121v1), 2025-02-24

(49) [WildFrame: Comparing Framing in Humans and LLMs on Naturally Occurring Texts](http://arxiv.org/abs/2502.17091v1), 2025-02-24

(50) [Automatically Evaluating the Paper Reviewing Capability of Large Language Models](http://arxiv.org/abs/2502.17086v1), 2025-02-24

(51) [Systematic Weight Evaluation for Pruning Large Language Models: Enhancing Performance and Sustainability](http://arxiv.org/abs/2502.17071v1), 2025-02-24

(52) [LLM-QE: Improving Query Expansion by Aligning Large Language Models with Ranking Preferences](http://arxiv.org/abs/2502.17057v1), 2025-02-24

(53) [Stable-SPAM: How to Train in 4-Bit More Stably than 16-Bit Adam](http://arxiv.org/abs/2502.17055v1), 2025-02-24

(54) [PrivaCI-Bench: Evaluating Privacy with Contextual Integrity and Legal Compliance](http://arxiv.org/abs/2502.17041v1), 2025-02-24

(55) [Understanding the Uncertainty of LLM Explanations: A Perspective Based on Reasoning Topology](http://arxiv.org/abs/2502.17026v1), 2025-02-24

(56) [Towards Auto-Regressive Next-Token Prediction: In-Context Learning Emerges from Generalization](http://arxiv.org/abs/2502.17024v1), 2025-02-24

(57) [Quantifying Logical Consistency in Transformers via Query-Key Alignment](http://arxiv.org/abs/2502.17017v1), 2025-02-24

(58) [Predicting Liquidity-Aware Bond Yields using Causal GANs and Deep Reinforcement Learning with LLM Evaluation](http://arxiv.org/abs/2502.17011v1), 2025-02-24

(59) [Be CIM or Be Memory: A Dual-mode-aware DNN Compiler for CIM Accelerators](http://arxiv.org/abs/2502.17006v1), 2025-02-24

(60) [An Enhanced Large Language Model For Cross Modal Query Understanding System Using DL-KeyBERT Based CAZSSCL-MPGPT](http://arxiv.org/abs/2502.17000v1), 2025-02-24

(61) [FADE: Why Bad Descriptions Happen to Good Features](http://arxiv.org/abs/2502.16994v1), 2025-02-24

(62) [Muon is Scalable for LLM Training](http://arxiv.org/abs/2502.16982v1), 2025-02-24

(63) [LongSafety: Evaluating Long-Context Safety of Large Language Models](http://arxiv.org/abs/2502.16971v1), 2025-02-24

(64) [Autoregressive Image Generation Guided by Chains of Thought](http://arxiv.org/abs/2502.16965v1), 2025-02-24

(65) [Make LLM Inference Affordable to Everyone: Augmenting GPU Memory with NDP-DIMM](http://arxiv.org/abs/2502.16963v1), 2025-02-24

(66) [UrduLLaMA 1.0: Dataset Curation, Preprocessing, and Evaluation in Low-Resource Settings](http://arxiv.org/abs/2502.16961v1), 2025-02-24

(67) [Lean and Mean: Decoupled Value Policy Optimization with Global Value Guidance](http://arxiv.org/abs/2502.16944v1), 2025-02-24

(68) [Reasoning Does Not Necessarily Improve Role-Playing Ability](http://arxiv.org/abs/2502.16940v1), 2025-02-24

(69) [BigMac: A Communication-Efficient Mixture-of-Experts Model Structure for Fast Training and Inference](http://arxiv.org/abs/2502.16927v1), 2025-02-24

(70) [FilterLLM: Text-To-Distribution LLM for Billion-Scale Cold-Start Recommendation](http://arxiv.org/abs/2502.16924v1), 2025-02-24

(71) [A Systematic Survey of Automatic Prompt Optimization Techniques](http://arxiv.org/abs/2502.16923v1), 2025-02-24

(72) [Benchmarking Temporal Reasoning and Alignment Across Chinese Dynasties](http://arxiv.org/abs/2502.16922v1), 2025-02-24

(73) [SPARC: Score Prompting and Adaptive Fusion for Zero-Shot Multi-Label Recognition in Vision-Language Models](http://arxiv.org/abs/2502.16911v1), 2025-02-24

(74) [AutoLogi: Automated Generation of Logic Puzzles for Evaluating Reasoning Abilities of Large Language Models](http://arxiv.org/abs/2502.16906v1), 2025-02-24

(75) [GuidedBench: Equipping Jailbreak Evaluation with Guidelines](http://arxiv.org/abs/2502.16903v1), 2025-02-24

(76) [Culture-TRIP: Culturally-Aware Text-to-Image Generation with Iterative Prompt Refinment](http://arxiv.org/abs/2502.16902v1), 2025-02-24

(77) [Char-mander Use mBackdoor! A Study of Cross-lingual Backdoor Attacks in Multilingual LLMs](http://arxiv.org/abs/2502.16901v1), 2025-02-24

(78) [Balancing Speech Understanding and Generation Using Continual Pre-training for Codec-based Speech LLM](http://arxiv.org/abs/2502.16897v1), 2025-02-24

(79) [Zero-shot Load Forecasting for Integrated Energy Systems: A Large Language Model-based Framework with Multi-task Learning](http://arxiv.org/abs/2502.16896v1), 2025-02-24

(80) [Unlocking Scientific Concepts: How Effective Are LLM-Generated Analogies for Student Understanding and Classroom Practice?](http://arxiv.org/abs/2502.16895v1), 2025-02-24

(81) [Make LoRA Great Again: Boosting LoRA with Adaptive Singular Values and Mixture-of-Experts Optimization Alignment](http://arxiv.org/abs/2502.16894v1), 2025-02-24

(82) [Applying LLMs to Active Learning: Towards Cost-Efficient Cross-Task Text Classification without Manually Labeled Data](http://arxiv.org/abs/2502.16892v1), 2025-02-24

(83) [DBudgetKV: Dynamic Budget in KV Cache Compression for Ensuring Optimal Performance](http://arxiv.org/abs/2502.16886v1), 2025-02-24

(84) [CORAL: Learning Consistent Representations across Multi-step Training with Lighter Speculative Drafter](http://arxiv.org/abs/2502.16880v1), 2025-02-24

(85) [A Multi-LLM-Agent-Based Framework for Economic and Public Policy Analysis](http://arxiv.org/abs/2502.16879v1), 2025-02-24

(86) [Graphy'our Data: Towards End-to-End Modeling, Exploring and Generating Report from Raw Data](http://arxiv.org/abs/2502.16868v1), 2025-02-24

(87) [Leveraging Large Language Models for Effective and Explainable Multi-Agent Credit Assignment](http://arxiv.org/abs/2502.16863v1), 2025-02-24

(88) [LongAttn: Selecting Long-context Training Data via Token-level Attention](http://arxiv.org/abs/2502.16860v1), 2025-02-24

(89) [Sarang at DEFACTIFY 4.0: Detecting AI-Generated Text Using Noised Data and an Ensemble of DeBERTa Models](http://arxiv.org/abs/2502.16857v1), 2025-02-24

(90) [Improving LLM General Preference Alignment via Optimistic Online Mirror Descent](http://arxiv.org/abs/2502.16852v1), 2025-02-24

(91) [Exploring Causes and Mitigation of Hallucinations in Large Vision Language Models](http://arxiv.org/abs/2502.16842v1), 2025-02-24

(92) ["Actionable Help" in Crises: A Novel Dataset and Resource-Efficient Models for Identifying Request and Offer Social Media Posts](http://arxiv.org/abs/2502.16839v1), 2025-02-24

(93) [REGen: A Reliable Evaluation Framework for Generative Event Argument Extraction](http://arxiv.org/abs/2502.16838v1), 2025-02-24

(94) [Finding the Sweet Spot: Preference Data Construction for Scaling Preference Optimization](http://arxiv.org/abs/2502.16825v1), 2025-02-24

(95) [Uncertainty Quantification of Large Language Models through Multi-Dimensional Responses](http://arxiv.org/abs/2502.16820v1), 2025-02-24

(96) [Grounded Persuasive Language Generation for Automated Marketing](http://arxiv.org/abs/2502.16810v1), 2025-02-24

(97) [CoT2Align: Cross-Chain of Thought Distillation via Optimal Transport Alignment for Language Models with Different Tokenizers](http://arxiv.org/abs/2502.16806v1), 2025-02-24

(98) [Multi-Agent Autonomous Driving Systems with Large Language Models: A Survey of Recent Advances](http://arxiv.org/abs/2502.16804v1), 2025-02-24

(99) [Unsupervised Topic Models are Data Mixers for Pre-training Language Models](http://arxiv.org/abs/2502.16802v1), 2025-02-24

(100) [AAD-LLM: Neural Attention-Driven Auditory Scene Understanding](http://arxiv.org/abs/2502.16794v1), 2025-02-24
